README.md

```diff
@@ -1,52 +1,62 @@
-`org.springframework.boot.autoconfigure.batch.JobLauncherCommandLineRunner` class to obtain all the parameters from
-command line. Almost all the parameters you can use are showed in the `example-launch.sh` script.
+Finally, before compiling the pipeline itself, you will need to clone and build its dependencies running these commands from a folder of your choice:
 
-### Examples
+```
+git clone https://github.com/EBIvariation/biodata.git
+cd biodata && mvn clean install -DskipTests
+cd ..
+git clone https://github.com/EBIvariation/opencga.git
+cd opencga && mvn clean install -DskipTests
+```
 
-Skeletons to load genotyped and aggregated VCF files are provided in the `examples` folder.
+### Build
 
-`application.properties` is used to configure database connections and applications the pipeline depends on: 
-[OpenCGA](https://github.com/opencb/opencga/tree/hotfix/0.5) and [Ensembl VEP](http://www.ensembl.org/info/docs/tools/vep/index.html).
+The latest stable version can be found in the [master](https://github.com/EBIvariation/eva-pipeline/tree/master) branch. [develop](https://github.com/EBIvariation/eva-pipeline/tree/develop) contains work in progress, which is fully tested but could be more unstable.
+
+If a MongoDB instance is available in the machine where you are running the build, you can test and build the application with `mvn test package`, otherwise please run `mvn package -DskipTests`.
+
+## Run
+
+Once successfully built, you can simply run the produced JAR file with `java -jar target/eva-pipeline-2.0-beta2-SNAPSHOT.jar`.
+
+Arguments to run the pipeline can be provided either using the command line or a properties file. Skeletons to load genotyped and aggregated VCF files are provided in the `examples` folder.
+
+`application.properties` is used to configure database connections and applications the pipeline depends on (OpenCGA and Ensembl VEP, see _Dependencies_ section).
 
 `load-genotyped-vcf.properties`, `load-aggregated-vcf.properties` and `initialize-database.properties` are job-specific configurations.
 
-If more convenient for your use case, the global and job configuration files can be merged into one.
+If more convenient for your use case, the global configuration and job parameters files can be merged into one.
 
-It is likely that you will need to change some parameters to fit your installation and/or or configure your job. For instance, 
-the location of your MongoDB databases, your OpenCGA/VEP installation directory, the folder were your files are, the type of job to run, etc.
+It is likely that you will need to edit some parameters to fit your installation and/or configure your job. For instance, 
+connection details to your MongoDB databases, OpenCGA/VEP installation directories, the folder were your files are, the type of job to run, etc. MongoDB connection details also need to be filled in the OpenCGA configuration file, located in `<OpenCGA root folder>/conf/storage-mongodb.properties`.
```

Comment:
I would clarify that the root folder is actually the `opencga-app/build` folder after compiling.